src/App.tsx

```diff
@@ -1,10 +1,54 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { FilteredBy } from './types/FilteredBy';
+import { getTodos } from './api/todos';
+import { Footer } from './Footer.tsx';
+import { TodoList } from './TodoList';
+import { Header } from './Header';
+import { Notifications } from './Notification';
+
+const USER_ID = 6459;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [todoStatus, setTodoStatus] = useState<FilteredBy>(FilteredBy.ALL);
+  const [errorFromServer, setErrorFromServer] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
```

Comment:
you can use one error state instead of 2. just check always if there is something in `errorMessage`
```suggestion
  const [errorMessage, setErrorMessage] = useState('');
```